README.md

```diff
@@ -155,20 +255,42 @@ Much more information on packaging Python software can be found in the [Hitchhik
+to your project about the degree of test coverage of your project.
 
-You will need a .travis.yml file in your repo. This file contains the configuration of your testing environment. This includes the different environments in which you will test the source code (for example, we test `shablona` against Python 2.7, Python 3.3 and Python 3.4). It includes steps that need to be taken before installation of the software. For example, installation of the software dependencies. For `shablona`, we use the [`Miniconda`](http://conda.pydata.org/miniconda.html) software distribution (not to be confused with [`Anaconda`](https://store.continuum.io/cshop/anaconda/), though they are similar and both produced by Continuum).
+You will need a .travis.yml file in your repo. This file contains the
+configuration of your testing environment. This includes the different
+environments in which you will test the source code (for example, we test
+`shablona` against Python 2.7, Python 3.3 and Python 3.4). It includes steps
+that need to be taken before installation of the software. For example,
+installation of the software dependencies. For `shablona`, we use the
+[`Miniconda`](http://conda.pydata.org/miniconda.html) software distribution (not
+to be confused with [`Anaconda`](https://store.continuum.io/cshop/anaconda/),
+though they are similar and both produced by Continuum).
 
-First, go to the Travis-CI [website](https://travis-ci.org/) and get a Travis user account, linked to your github user account.
+First, go to the Travis-CI [website](https://travis-ci.org/) and get a Travis
+user account, linked to your github user account.
 
-You will need to set up your github repo to talk to Travis (More explanation + pictures will come here).
+You will need to set up your github repo to talk to Travis (More explanation +
+pictures will come here).
 
-You will need to go back to travis-ci, and flip on the switch on that side as well.
+You will need to go back to travis-ci, and flip on the switch on that side as
+well.
 
-The travis output will also report to you about test coverage, if you set it up that way.
+The travis output will also report to you about test coverage, if you set it up
+that way.
 
-You will start getting emails telling you the state of the testing suite on every pull request for the software, and also when you break the test suite on the `master` branch. That way, you can be pretty sure that the `master` is working (or at least know when it isn't...).
+You will start getting emails telling you the state of the testing suite on
+every pull request for the software, and also when you break the test suite on
+the `master` branch. That way, you can be pretty sure that the `master` is
+working (or at least know when it isn't...).
 
+To also continuously test your code on a Windows system, you can also add a
+`.appveyor` file and activate your repository on [Appveyor](http://www.appveyor.com/).  
```

Comment:
Should this be `appveyor.yml`? Also, you go into much less detail about the setup than you do for Travis (just a note)